aws/cloud/master-compliance-test.json

```diff
@@ -21135,6 +21135,756 @@
                             ]
                         }
                     ]
+                },
+                {
+                    "masterTestId": "PR-AWS-CLD-IAM-051",
+                    "masterSnapshotId": [
+                        "TEST_IAM_01"
+                    ],
+                    "type": "rego",
+                    "rule": "file(iam.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AWS-CLD-IAM-051",
+                            "eval": "data.rule.managed_iam_policy_dont_have_full_s3_action_permission",
+                            "message": "data.rule.managed_iam_policy_dont_have_full_s3_action_permission_err",
+                            "remediationDescription": "",
+                            "remediationFunction": "PR_AWS_CLD_IAM_051.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "Ensure AWS IAM managed policies do not have 'getObject' or full S3 action permissions",
+                    "description": "Ensuring AWS IAM managed policies do not have 'getObject' or full S3 action permissions, prevents potential Amazon S3 data exfiltration or manipulation.",
+                    "tags": [
+                        {
+                            "cloud": "AWS",
+                            "compliance": [
+                            ],
+                            "service": [
+                                "iam"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "masterTestId": "PR-AWS-CLD-IAM-028",
+                    "masterSnapshotId": [
+                        "TEST_IAM_06"
+                    ],
+                    "type": "rego",
+                    "rule": "file(iam.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AWS-CLD-IAM-028",
+                            "eval": "data.rule.iam_access_key_disabled_on_root_account",
+                            "message": "data.rule.iam_access_key_disabled_on_root_account_err",
+                            "remediationDescription": "",
+                            "remediationFunction": "PR_AWS_CLD_IAM_028.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "AWS root account should not use access keys",
+                    "description": "To strengthen the security of your AWS environment and adhere to IAM best practices, ensure that the AWS root account user doesn't use access keys for making API requests to access cloud resources or billing information.",
+                    "tags": [
+                        {
+                            "cloud": "AWS",
+                            "compliance": [
```

Comment:
@rezoan  please insert all the related compliance framework

Reply:
@farchide compliance added.